flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBConfigurableOptions.java

```diff
@@ -127,6 +127,14 @@
 			.withDescription("The approximate size (in bytes) of user data packed per block. " +
 				"RocksDB has default blocksize as '4KB'.");
 
+	public static final ConfigOption<MemorySize> METADATA_BLOCK_SIZE =
+		key("state.backend.rocksdb.block.metadata-blocksize")
+			.memoryType()
+			.noDefaultValue()
+			.withDescription("Approximate size (in bytes) of partitioned metadata packed per block. " +
```

Comment:
As we use `MemorySize` to wrap this option, we could no longer need to describe it as `in bytes`.